<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Icons/Icons/file.ico</normaloff>:/Icons/Icons/file.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>ESP File system</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QListWidget" name="listFilesESP"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="btnContent">
                 <property name="text">
                  <string>Show content</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnRemove">
                 <property name="text">
                  <string>Remove</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnReset">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Scripts </string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QListWidget" name="listScripts"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="btnSetMain">
                 <property name="text">
                  <string>Main</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnUploadESP">
                 <property name="text">
                  <string>Upload</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnProgram">
                 <property name="text">
                  <string>Program</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Modules</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QListWidget" name="listModule"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnUploadModule">
               <property name="text">
                <string>Upload</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Content files</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QTextEdit" name="txtbox"/>
           </item>
           <item>
            <widget class="QTextEdit" name="txtConsole"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QProgressBar" name="progressProgramming">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblProgram">
            <property name="text">
             <string>Waiting</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionScript_folder"/>
    <addaction name="actionLibrary_folder"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionScript_folder">
   <property name="text">
    <string>Script folder</string>
   </property>
  </action>
  <action name="actionLibrary_folder">
   <property name="text">
    <string>Module folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
